core/server/api_container/server/startosis_engine/builtins/kurtosis_module.go

```diff
@@ -15,6 +15,11 @@ const (
 	connectionSubmoduleAllowedAttrName = "ALLOWED"
 )
 
+var (
+	totalPacketLossPercentage = starlark.Float(100) //nolint:gomnd
+	noPacketLossPercentage    = starlark.Float(0)
```

Comment:
possibly nit:

packageLostPercentage = 100

and in the code we could just do starlark.Float(packageLostPercentage)

also why don't we need `//nolint` on line `20`

Reply:
Done!